src/Security/SecProtocolMetadata.cs

```diff
@@ -176,54 +179,54 @@ public class SecProtocolMetadata : NativeObject {
 
 		public bool EarlyDataAccepted => sec_protocol_metadata_get_early_data_accepted (GetCheckedHandle ()) != 0;
 
- 		[DllImport (Constants.SecurityLibrary)]
+		[DllImport (Constants.SecurityLibrary)]
 		[return: MarshalAs (UnmanagedType.I1)]
 		extern static bool sec_protocol_metadata_challenge_parameters_are_equal (IntPtr metadataA, IntPtr metadataB);
 
 		public static bool ChallengeParametersAreEqual (SecProtocolMetadata metadataA, SecProtocolMetadata metadataB)
 		{
-			if (metadataA == null)
-				return metadataB == null;
-			else if (metadataB == null)
+			if (metadataA is null)
+				return metadataB is null;
+			else if (metadataB is null)
 				return false; // This was tested in a native app. We do copy the behaviour.
 			return sec_protocol_metadata_challenge_parameters_are_equal (metadataA.GetCheckedHandle (), metadataB.GetCheckedHandle ());
 		}
 
- 		[DllImport (Constants.SecurityLibrary)]
+		[DllImport (Constants.SecurityLibrary)]
 		[return: MarshalAs (UnmanagedType.I1)]
 		extern static bool sec_protocol_metadata_peers_are_equal (IntPtr metadataA, IntPtr metadataB);
 
 		public static bool PeersAreEqual (SecProtocolMetadata metadataA, SecProtocolMetadata metadataB)
 		{
-			if (metadataA == null)
-				return metadataB == null;
-			else if (metadataB == null)
+			if (metadataA is null)
+				return metadataB is null;
+			else if (metadataB is null)
 				return false; // This was tested in a native app. We do copy the behaviour.
 			return sec_protocol_metadata_peers_are_equal (metadataA.GetCheckedHandle (), metadataB.GetCheckedHandle ());
 		}
 
 		delegate void sec_protocol_metadata_access_distinguished_names_handler_t (IntPtr block, IntPtr dispatchData);
- 		static sec_protocol_metadata_access_distinguished_names_handler_t static_DistinguishedNamesForPeer = TrampolineDistinguishedNamesForPeer;
-
- 		[MonoPInvokeCallback (typeof (sec_protocol_metadata_access_distinguished_names_handler_t))]
- 		static void TrampolineDistinguishedNamesForPeer (IntPtr block, IntPtr data)
- 		{
- 			var del = BlockLiteral.GetTarget<Action<DispatchData>> (block);
- 			if (del != null) {
- 				var dispatchData = new DispatchData (data, owns: false);
- 				del (dispatchData);
- 			}
- 		}
-
- 		[DllImport (Constants.SecurityLibrary)]
```

Comment:
Do I agree with the re-indentation, yes. Does it make the review harder, yes. Can we do the reindentation after.